sidebars.js

```diff
@@ -49,6 +49,12 @@ const sidebars = {
       label: '🔧 API Docs',
       className: 'category-link',
     },
+    {
+      type: 'link',
+      href: '/advanced',
+      label: '🛰 Advanced',
```

Comment:
Does it make sense to make the title a little more descriptive? Something like "Advanced Usage"?